starknet_py/tests/e2e/client/conftest.py

```diff
@@ -11,14 +11,15 @@
 from starknet_py.net.full_node_client import FullNodeClient
 from starknet_py.net.gateway_client import GatewayClient
 from starknet_py.net.models import StarknetChainId
+from starknet_py.tests.e2e.conftest import directory_with_contracts
 
 directory = os.path.dirname(__file__)
 
 
 def prepare_devnet(net: str) -> dict:
-    script_path = Path(directory, "prepare_devnet_for_gateway_test.sh")
-    contract_compiled = Path(directory, "balance_compiled.json")
-    contract_abi = Path(directory, "balance_abi.json")
+    script_path = Path(directory + "/prepare_devnet_for_gateway_test.sh")
```

Comment:
these days i tend to prefer this style of merging paths:
`    script_path = Path(directory) / "prepare_devnet_for_gateway_test.sh"`